docs/fidesops/docs/guides/fidesops_workflow.md

```diff
@@ -1,206 +1,130 @@
-execution to exit with a status of `paused`. Request execution can be continued when ready using a token that was 
-supplied in the original request.
+### Access request automation
+Access request automation is performed regardless of whether there are access or erasure Rules defined, as both Rules require this data.  See how to [configure policies, rules, and rule targets](policies.md) for additional information. 
 
-No data uploaded by Policy Webhooks is returned to the data subject, but identities discovered can be used 
-to later locate data pertaining to the subject in the Access Request Step.
-
-If a request to a pre-execution webhook fails, request execution will exit with a status of `error`. Retrying 
-the privacy request will resume from this step and attempt to re-run all the pre-execution webhooks.
-
-
-### 3) Access Request Step
-
-The automated Access Request Step is performed regardless of whether there are Access or Erasure Rules defined, since
-both Rules require this data.  See here to [Configure Policies, Rules, and Rule Targets](policies.md). This step visits all Collections and retrieves
-all Fields that you've defined in your [Datasets](datasets.md). We build a graph in accordance with how you've designated your Collections are related, 
-visit each Collection in turn, and end by gathering all the results together.
+This step visits all Collections and retrieves all Fields that you've defined in your [Datasets](datasets.md). Fidesops builds a graph in accordance with how you've designated your Collections are related, visits each Collection in turn, and gathers all the results together.
 
 #### Graph building
-Fidesops builds a DAG (Directed Acyclic Graph) from all Collections within your enabled Datasets. In other words, each location or node on the graph 
-corresponds to a Collection in one of your Datasets. The graph helps determine the order in which nodes will be visited.
-We start with any Collections that we can query just using the identity data, and then point those Collections toward
-dependent Collections that need data from the current Collection and so on. If fidesops can't figure out how to 
-reach a Collection, it will exit early with a status of `error`.  To remedy, you will have to
-how your Collections are related to each other in your Datasets and resubmit the privacy request.
+Fidesops builds a Directed Acyclic Graph, or DAG, where each location or node corresponds to a Collection in one of your Datasets. The graph helps determine the order in which nodes will be visited. Fidesops begins with any Collections that can be queried using the supplied identity data, and then points those Collections toward dependent Collections, etc. If fidesops can't determine out how to reach a Collection, it will exit early with a status of `error`.  To remedy an errored access request, you update how your Collections are related to each other in your Datasets, and resubmit the privacy request. 
+
```

Comment:
Just checking, is `determine out how to reach` intentional or would it be `determine how to reach`